reactor-core/src/blockHoundTest/java/reactor/core/scheduler/BoundedElasticSchedulerBlockhoundTest.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2011-Present VMware Inc. or its affiliates, All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package reactor.core.scheduler;
+
+import java.util.concurrent.FutureTask;
+
+import org.junit.Test;
+
+import reactor.blockhound.BlockHound;
+import reactor.core.Disposable;
+import reactor.core.Disposables;
+
+/**
+ * @author Simon Baslé
+ */
+public class BoundedElasticSchedulerBlockhoundTest {
+
+	static {
+		// Use the builder to load only our integration to avoid false positives
+		BlockHound.builder()
```

Comment:
`BlockHound.install()` would be fine here

Reply:
it would be at odds with ReactorBlockHoundIntegrationTest's own usage of the `Builder`, wouldn't it?

Reply:
you can (and should) configure it to isolate test runs by setting `forkEvery = maxParallelForks = 1` on blockHoundTest task, similar to how we do it in BlockHound:
https://github.com/reactor/BlockHound/blob/b9c60096113bba2448fe00d90eff791de14495a2/example/build.gradle#L17